### ISSUE TITLE
Morocco data Incoherence between arcgis & github all based on JHU

### ISSUE THREAD

**Issue by kbaina** [CSSEGISandData/COVID-19#1017]
Dear all,

Thank you for your efforts.

Morocco data is false for Marsh 18.

The same central Johns Hopkins University data set  is used by https://storymaps.arcgis.com/stories/4fdc0d03d3a34aa485de1fb0d2650ee0.

It shows 54 (official number) for Morocco and not 49 as shown in github.

There is a incoherence some where ?

@kbaina

_Originally posted by @kbaina in https://github.com/CSSEGISandData/COVID-19/issues/994#issuecomment-600987707_

**Comment by zakblack** [comment]
Yeah and the problem still unsolved, Today its 61 cases and it shows 54

**Comment by cipriancraciun** [comment]
Today another issue was opened about the Morocco data: #1682 ;  if this ticket is still valid, and in order to help the JHU team, would you contribute your findings to that ticket?  (If so please close this ticket in order to keep things tidy.)
